lib/stPacket/verification/verifySTPacketFactory.js

```diff
@@ -64,6 +68,23 @@ function verifySTPacketFactory(verifyContract, verifyDocuments, dataProvider) {
       result.merge(
         await verifyDocuments(stPacket, userId, contract),
       );
+
+      const dataTriggersExecutionContext = new DataTriggerExecutionContext(
```

Comment:
Does it make sense to call triggers if we have invalid documents?

Reply:
Yeah :+1: will fix